.github/workflows/pip-compile-docs.yml

```diff
@@ -37,3 +33,5 @@ jobs:
       pr-branch: "${{ inputs.pr-branch || 'pip-compile/devel/docs' }}"
       nox-args: "-e 'pip-compile-3.10(requirements)' 'pip-compile-3.10(requirements-relaxed)'"
       reset-branch: "${{ inputs.reset-branch || false }}"
+    secrets: inherit
+    environment: github-bot
```

Comment:
@gotmax23 why is this needed on the calling side?

Reply:
This is apparently needed when using secrets with a reusable workflow.